doc/content/integrations/mqtt/Connecting to Node-RED/_index.md

```diff
@@ -15,50 +17,69 @@ First, you need to run the Node-RED by using the following command in terminal:
+{{< figure src="mqtt_in_node_properties.png" alt="mqtt in node properties" >}}
 
 In the **Server** dropdown menu, select **Add new mqtt-broker** and click on the button besides to edit it. 
 
-{{< figure src="node_connection.png" alt="COnfiguring MQTT Server connection information" >}}
+{{< figure src="mqtt_in_node_connection.png" alt="COnfiguring MQTT Server connection information" >}}
 
-In the **Connection** tab, under **Server**, you should provide MQTT server's URL - if it runs locally, then type **localhost**. The **Port** value depends on whether the connection is to be set up as a secure one or not. In this example, we are using TLS, so the **Port** value is set to 8883. In this case you should also check the **Enable secure (SSL/TLS) connection** box.
+In the **Connection** tab, under **Server**, you should provide MQTT Server's URL - if it runs locally, then type **localhost**. The **Port** value depends on whether the connection is to be set up as a secure one or not. In this example, TLS is being used, so the **Port** value is set to 8883. In this case you should also check the **Enable secure (SSL/TLS) connection** box.
 
-{{< figure src="node_security.png" alt="Configuring MQTT Server credentials" >}}
+{{< figure src="mqtt_in_node_security.png" alt="Configuring MQTT Server credentials" >}}
 
-In the **Security** tab, enter the **Username** and **Password** according to values in the Console.
+In the **Security** tab, enter the **Username** and **Password** according to values in the in the {{% tts %}} Console.
 
-Under **Topic**, a topic to listen to can be specified, but for the testing purposes set the value to `#` (all topics). The **QoS** value can be arbitrarily chosen, as well as the **Output**. In this example, we want to see each message as `a parsed JSON object`. 
+Under **Topic**, a topic to listen to can be specified, but for the testing purposes set the value to `#` (all topics). The **QoS** value can be selected from the listed options, as well as the **Output**. In this example, we want to see each message as `a parsed JSON object`. 
 
 >Note: instead of changing **Output** parameter to `a parsed JSON object`, you can also use a `JSON` node or the `function` node with proper parsing function.
 
-Next, add two `debug` nodes to the flow and connect both of them to the `mqtt in` node. One debug node will listen to the events, while the other will listen to messages coming from the devices. To make this happen, set the **Output** parameter for the first debug node configuration to `complete msg object` value, and to `msg.payload` for the other. 
+Next, add two **debug** nodes to the flow and connect both of them to the **mqtt in** node. One debug node will listen to the events, while the other will listen to messages coming from the devices. To make this happen, set the **Output** parameter for the first debug node configuration to `complete msg object` value, and to `msg.payload` for the other. 
```

Comment:
>will listen to messages coming from the devices

This is not entirely true if you subscribe to `#` as you get a lot of other events which are not messages from end devices; those would only be `v3/{application id}/devices/{device id}/up`